### PR TITLE
Wt.neighborhoods to site

### PR DESCRIPTION
**Pull request by willmt80**
Clickup ticket: https://app.clickup.com/t/1dk47gz

Added neighborhood_id to the site table with a migration script. Ran mvn clean install to generate new site record files. Updated the getSite request to include the neighborhood name.

neighborhood

### CODE REVIEW

File: service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java [hunk 1]
```diff
@@ -133,6 +134,12 @@ public GetSiteResponse getSite(int siteId) {
       throw new ResourceDoesNotExistException(siteId, "site");
     }
 
+    String neighborhood =
+        db.selectFrom(NEIGHBORHOODS)
+            .where(NEIGHBORHOODS.ID.eq(sitesRecord.getNeighborhoodId()))
+            .fetchOne()
```
Comment (connernilsen) [inline service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java:140]: This can't be null, right? Just want to double check

**Review by connernilsen — APPROVED** [review]
Looks great!

**Review by vrushalitarte — APPROVED** [review]
looks good to me!

**Review by chromium-52 — APPROVED** [review]
Looks great!

File: service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java (hunk 1 above)
Comment (willmt80) [reply service/src/main/java/com/codeforcommunity/processor/SiteProcessorImpl.java:140]: Correct, this is a required field

**Pull request merged**